### PR TITLE
update documentation of property comment sniff

### PR DESCRIPTION
**Pull request by xalopp**

### CODE REVIEW

**Comment by xalopp** [comment]
@Ma27 👓 ! 😸

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
 * interpolated variables in double quoted strings must be surrounded by `{ }`, e.g. `{$VAR}` instead of `$VAR`
 * `sprintf` or `"{$VAR1} {$VAR2}"` must be used instead of the dot operator; concat operators are only allowed to concatenate constants and multi line strings,
 * a whitespace is required after each typecast, e.g. `(int) $value` instead of `(int)$value`
-
+* doc blocks of class properties must be multiline and have one `@var` annotation
```
Comment (mmoll) [inline README.md]: maybe "...have exactly one..."

**Comment by Ma27** [comment]
agree with @mmoll regarding "have exactly one", everything else looks fine to me :)

**Comment by xalopp** [comment]
@mmoll @Ma27 the "have exactly one var annotation" thing is changed 😺

**Pull request merged**